src/sidebar/util/version-data.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+/**
+ * An object representing user info
+ *
+ * @typedef {Object} UserInfo
+ * @property {string=} userid
+ * @property {string=} displayName
+ */
+
+/**
+ * An object representing document metadata
+ *
+ * @typedef {Object} DocMetadata
+ * @property {string=} documentFingerprint - optional PDF fingerprint for
+ *                     current document
+ */
+
+/**
+ * An object representing document info
+ *
+ * @typedef {Object} DocumentInfo
+ * @property {string=} url - current document URL
+ * @property {DocMetadata} metadata - document metadata
+ */
+
+class VersionData {
+  /**
+   * @param {UserInfo} userInfo
+   * @param {DocumentInfo} documentInfo
+   * @param {Window} window_ - test seam
```

Comment:
This is an affordance that allows for dependency injection here such that tests can provide their own `Window` object (instead of the `util` using the globally-available `window`).